pkg/api/api.go

```diff
@@ -417,7 +417,7 @@ func (hs *HTTPServer) registerRoutes() {
 			alertsRoute.Get("/states-for-dashboard", routing.Wrap(hs.GetAlertStatesForDashboard))
 		})
 
-		apiRoute.Get("/alert-notifiers", reqEditorRole, routing.Wrap(
+		apiRoute.Get("/alert-notifiers", reqSignedIn, routing.Wrap(
```

Comment:
This change has been introduced because we use this endpoint in the `Contact points` list view to show the correct notifier _name_ instead of the _type_.

We don't think this has any security ramifications but will double check with the right stakeholders.